scripts/autotools/libmongoc/CheckSasl.m4

```diff
@@ -1,12 +1,18 @@
 PHP_ARG_WITH([mongodb-sasl],
              [whether to enable SASL for Kerberos authentication],
-             [AS_HELP_STRING([--with-mongodb-sasl=@<:@auto/no/cyrus@:>@],
+             [AS_HELP_STRING([--with-mongodb-sasl=@<:@auto/cyrus/no@:>@],
```

Comment:
For consistency, I lead with "auto" and end with "no" in option help text (and the PECL prompts).